flutter/shell/platform/tizen/tizen_renderer_nui_gl.cc

```diff
@@ -0,0 +1,71 @@
+// Copyright 2025 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/tizen/tizen_renderer_nui_gl.h"
+#include "flutter/shell/platform/tizen/tizen_renderer_egl.h"
+#include "tizen_renderer.h"
+
+namespace flutter {
+
+TizenRendererNuiGL::TizenRendererNuiGL(TizenViewNui* view_nui,
```

Comment:
I have a question. 
Why did you make this class (TizenRendererNuiGL)?
It just seems to wraps TizenRendererEgl.
Could you please explain what your plans are?